README.md

```diff
@@ -253,7 +253,7 @@ status:
     state: ready
   - name: state-SRIOV-device-plugin
     state: ignore
-  - name: state-RDMA-device-plugin
+  - name: state-RDMA-shared-device-plugin
```

Comment:
im fine with struct name changes in code. but not with this one.

thats user facing, probably testing teams rely on it.

Reply:
Yes, you are totally right. That change was only on purpose, I had actively decided against it. Will revert it

Reply:
reverted